### DISCUSSION TITLE
Being able to use local models like llama 3.0

### DISCUSSION THREAD

**Discussion by GJoe2** [lifan0127/ai-research-assistant#108] [Ideas]
Would be great and nice to have for people who works with local models, [Ollama](https://github.com/ollama/ollama) made a great cli software.

**Comment by akportella** [comment]
Yes, that would be great! I've just tried a few questions, got some wrong and incomplete answers and it cost me 10 cents ! For my real workload I project I would spent like $1.00 per day, or $20 per month. Schorlacy custs ~10 dollar a month and It is much more complete and it is integrated with Zotero as well.

**Comment by promedacademy** [comment]
@lifan0127  @caomingpei Nomic GPT4all and anything and LM studio offer local server with OpenAI compatible prompt and api , so it would be great to add the option to add the URL of the endpoint of the local server running LLM . first it will be totally private , second it will be free and save alot of money
